test/compress/global_defs.js

```diff
@@ -123,6 +126,8 @@ mixed: {
         console.log(CONFIG);
     }
     expect: {
+        const FOO = { BAR: 0 };
+        console.log(FOO.BAR);
```

Comment:
If `FOO` is global - which it is in this test case - then `FOO.BAR` should be changed to `0`.